lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -453,7 +453,7 @@ sub mark_job_linked {
     return undef if !$job || ($referer->path_query =~ /^\/?$/);
     my $comments = $job->comments;
     return undef if $comments->find({text => {like => 'label:linked%'}});
-    my $user = $self->result_source->schema->resultset('Users')->search({username => 'system'})->first;
+    my $user = $self->result_source->schema->resultset('Users')->find({username => 'system'}, {select => ['id']});
```

Comment:
see https://github.com/os-autoinst/openQA/pull/4202#discussion_r708957986

Reply:
Ah right, I remember that discussion.
Then, the correct thing would be to also specify `provider` (as you wrote yourself).
So I added `provider => ''`.

Reply:
good